snuba/migrations/operations.py

```diff
@@ -52,9 +53,19 @@ def get_nodes(self) -> Sequence[ClickhouseNode]:
             cluster.get_distributed_nodes(),
         )
 
+        cluster_name = f"{cluster.get_host()}:{cluster.get_port()}"
+        readable_name = cluster.get_clickhouse_cluster_name()
```

Comment:
do we allow this to be empty? is `cluster_name` only used for logs?

Reply:
we allow this to be optional, I'm not sure why